src/variety/refactor.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
I don't think any of these shell files need to be here.

Reply:
oops, didn't mean to commit those